javascript/app-model-exposure/src/pages/DefineDomainModelPage.hooks.tsx

```diff
@@ -5,17 +5,47 @@
 
 import type { IChoiceGroupOption } from '@fluentui/react'
 import type { Hypothesis } from '@showwhy/app-common'
+import { useDebounceFn } from 'ahooks'
 import { type FormEvent, useCallback } from 'react'
+import { v4 as uuiv4 } from 'uuid'
 
 import { useSetCausalQuestion } from '../state/causalQuestion.js'
+import { useDefinitionsState } from '../state/definitions.js'
+import { CausalityLevel } from '../types/causality/CausalityLevel.js'
+import type { DefinitionType } from '../types/experiments/DefinitionType.js'
 import type { Maybe } from '../types/primitives.js'
 import type { CausalQuestion } from '../types/question/CausalQuestion.js'
 import type { CausalQuestionElement } from '../types/question/CausalQuestionElement.js'
 
+function useAddDefaultDefinition() {
+	const [definitions, setDefinitions] = useDefinitionsState()
+	return useDebounceFn(
```

Comment:
Why is this debounced?

Reply:
to wait a sec for the user to stop typing and add that new label as an alternative definition